components/Sections/AboutSection.tsx

```diff
@@ -19,24 +19,28 @@ const AboutSection: FC<IProps> = ({ isMobile }) => {
         <div className="content-container">
           <h1>About DeerHacks</h1>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tortor quam, pulvinar a
-            neque ut, volutpat placerat ligula. Sed ac tellus eget magna pretium finibus. Praesent
-            porta massa vel quam pellentesque molestie. Vestibulum ante ipsum primis in faucibus
-            orci luctus et ultrices posuere cubilia curae; Vestibulum ut nisl elit.
+            For the 1st time in UTM, DeerHacks, will bring together 500 innovators,
```

Comment:
```suggestion
            For the 1st time in UTM, DeerHacks, will bring together numerous innovators,
```
Let's not put a number here for now